### PR TITLE
Update abc templates

### PR DESCRIPTION
**Pull request by lock14**
Updated the templates to use the CLI command syntax changes.

### CODE REVIEW

**Comment by lock14** [comment]
This PR might not be able to be merged until we actually do the `1.0.0` release.

File: abc.templates/base-workflows/contents/guardian-admin.yml [hunk 1]
```diff
@@ -129,11 +127,9 @@ jobs:
...
-          TERRAFORM_ARGS: '${{ inputs.args }}'
           DIRECTORY: '${{ matrix.working_directory }}'
         shell: 'bash'
         run: |-
           guardian run \
-          -terraform-command="${TERRAFORM_COMMAND}" \
-          -terraform-args="${TERRAFORM_ARGS}" \
-          "${DIRECTORY}"
+          -dir="${DIRECTORY}" \
+          -- ${TERRAFORM_COMMAND}"
```
Comment (lock14) [inline abc.templates/base-workflows/contents/guardian-admin.yml:135]: I did some testing and this should not be subject to any injection type attacks.
For example if the user typed in `; echo 'hello'` then each 'word' would get passed in as guardian command line args. i.e. guardian's args would be `["run", "-dir=directory", "--", ";", "echo", "'hello'"]` which would simply lead to an error when the args  `[";", "echo", "'hello'"]` get passed along to the terraform binary.

**Comment by lock14** [comment]
> This PR might not be able to be merged until we actually do the `1.0.0` release.

actually I think we would want to merge this  in prior to doing the `1.0.0` tag so that these new templates are included in that release.

File: abc.templates/base-workflows/contents/guardian-admin.yml (hunk 1 above)
Comment (verbanicm) [reply abc.templates/base-workflows/contents/guardian-admin.yml:135]: Great thank you!

Comment (verbanicm) [inline abc.templates/base-workflows/contents/guardian-admin.yml:135]: Looks like there's an extra " at the end here

File: abc.templates/base-workflows/contents/guardian-plan.yml [hunk 2]
```diff
@@ -72,7 +72,7 @@ jobs:
           SOURCE_REF: '${{ github.event.pull_request.base.sha }}'
           DEST_REF: '${{ github.event.pull_request.head.sha }}'
         run: |-
-          DIRECTORIES=$(guardian entrypoints -detect-changes -source-ref="${SOURCE_REF}" -dest-ref="${DEST_REF}" -format=json REPLACE_TERRAFORM_DIRECTORY)
+          DIRECTORIES=$(guardian entrypoints -dir=REPLACE_TERRAFORM_DIRECTORY -detect-changes -source-ref="${SOURCE_REF}" -dest-ref="${DEST_REF}" -format=json)
```
Comment (verbanicm) [inline abc.templates/base-workflows/contents/guardian-plan.yml:75]: nit: can we wrap the dir in double quotes?

File: abc.templates/base-workflows/contents/guardian-apply.yml [hunk 3]
```diff
@@ -63,7 +63,7 @@ jobs:
         id: 'dirs'
         shell: 'bash'
         run: |-
-          DIRECTORIES=$(guardian entrypoints -detect-changes -source-ref=HEAD~1 -dest-ref=HEAD -format=json REPLACE_TERRAFORM_DIRECTORY)
+          DIRECTORIES=$(guardian entrypoints -dir=REPLACE_TERRAFORM_DIRECTORY -detect-changes -source-ref=HEAD~1 -dest-ref=HEAD -format=json)
```
Comment (verbanicm) [inline abc.templates/base-workflows/contents/guardian-apply.yml:66]: nit: can we wrap the dir in double quotes?

File: abc.templates/base-workflows/contents/guardian-admin.yml (hunk 1 above)
Comment (lock14) [reply abc.templates/base-workflows/contents/guardian-admin.yml]: fixed.

File: abc.templates/base-workflows/contents/guardian-plan.yml (hunk 2 above)
Comment (lock14) [reply abc.templates/base-workflows/contents/guardian-plan.yml]: done

File: abc.templates/base-workflows/contents/guardian-apply.yml (hunk 3 above)
Comment (lock14) [reply abc.templates/base-workflows/contents/guardian-apply.yml]: done

**Review by verbanicm — APPROVED** [review]

**Pull request merged**